addons/account/models/account_journal_dashboard.py

```diff
@@ -664,6 +723,52 @@ def action_create_vendor_bill(self):
             'views': [[view_id, 'form']],
         }
 
+    def action_open_payment_items(self, options=None):
+        action = self.env['ir.actions.actions']._for_xml_id('account.action_open_payment_items')
+        if options:
+            domain = ast.literal_eval(action['domain'])
+            if options.get('domain'):
+                options_domain = ast.literal_eval(options.get('domain'))
+                domain += options_domain
+            model_info = self._get_model_info_from_id(options['line']['id'])
+            if model_info[0] == 'res.partner':
+                domain.append(('partner_id', '=', model_info[1]))
+            elif model_info[0] == 'account.move.line':
+                domain.append(('move_name', '=', self.env[model_info[0]].search([('id', '=', model_info[1])]).move_name))
+
+            if options.get('period'):
+                domain.extend(
+                    self.env['account.report'].build_domain_from_period(options.get('period'))
```

Comment:
This is still not defined in community

Does any extension from here (the `options` management) actually need to be in community?
It all seems only related to `account_reports`, keep it in `account_reports`